.github/workflows/nodejs.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Setup npm version 7
+        run: npm i -g npm@7 --registry=https://registry.npmjs.org
```

Comment:
Is setting registry required?

Reply:
Not sure, I need to do some more testing for caching